Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Hey Tom. Please use the new Setup cell that does a much neater job of the first 90% of this cell. Have a look at the notebooks in this PR https://github.com/Azure/Azure-Sentinel-Notebooks/pull/29</p><p>You can also use the blurb in the markdown cell immediately before this (or massage to fit in with your style)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I think Pete also covered this. I have an update in PR, as I said above.</p>

Reply:
<p>Thanks Ian, I have adopted the new setup cell and blurb!</p>